brats/transformations.py

```diff
@@ -73,3 +78,77 @@ def hist_match(source: np.ndarray, template: np.ndarray) -> np.ndarray:
                                     template_values)
         result[positive] = interp_t_values[bin_idx]
         return result
+
+
+class NiftiOrderTransformation:
+    """
+    Changes dimensions order from nifti (H,W,D,C) to torch convention (C,D,W,H).
+    It can be applied to both ``torch.Tensor`` or ``numpy.ndarray``.
+    """
+
+    def __call__(self, img: typing.Union[np.ndarray, torch.Tensor]) -> typing.Union[np.ndarray, torch.Tensor]:
+        assert len(img.shape) == 4
+
+        if isinstance(img, torch.Tensor):
+            transformed = img.permute(3, 2, 0, 1)
+        if isinstance(img, np.ndarray):
+            transformed = np.moveaxis(img, [0, 1, 2, 3], [2, 3, 1, 0])
+        return transformed
+
+
+class AddChannelDimToMaskTransformation:
+    """
+    Adds additional channel dimension
+    It can be applied to both ``torch.Tensor`` or ``numpy.ndarray``.
+    """
+
+    def __call__(self, img: typing.Union[np.ndarray, torch.Tensor]) -> typing.Union[np.ndarray, torch.Tensor]:
+        assert len(img.shape) == 3
+
+        if isinstance(img, torch.Tensor):
+            transformed = torch.unsqueeze(img, 3)
+        if isinstance(img, np.ndarray):
+            transformed = np.expand_dims(img, 3)
+        return transformed
+
+
+class BinarizationTransformation:
+    """
+    Adds additional channel dimension
+    It can be applied to``numpy.ndarray``.
+    """
+
+    def __call__(self, img: typing.Union[np.ndarray, torch.Tensor]) -> typing.Union[np.ndarray, torch.Tensor]:
+        img[img > 0] = 1
+        return img
+
+
+class ResizeVolumeTransformation:
+    """
+    Resizes the volume spatial dimensions (H,W) to the given size.
+    For detail, see nn.functional.interpolate, this class is only
+    """
+
+    def __init__(self, size: typing.Union[int, tuple]):
+        self.size = size
+
+    def __call__(self, tensor: torch.Tensor) -> torch.Tensor:
+        assert len(tensor.shape) == 4
```

Comment:
As above.

Reply:
Also can be change to `tensor.dimensions() == 4` or `tensor.ndim == 4` or semething else depending on the version of pytorch.

Reply:
changed